pkg/systemd/timer-dialog.jsx

```diff
@@ -215,6 +213,7 @@
                                 <FormSelectOption value="weeks" label={_("Weeks")} />
                             </FormSelect>
                         </Flex>
+                        <FormHelper helperTextInvalid={submitted && validationFailed.delayNumber && _("Delay must be a number")} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18641-20230417-051207-efb10d39-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)